weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -1323,21 +1327,38 @@ const isValidTraceCall = (callRes: traceServerClient.TraceCallSchema) => {
   return !('detail' in callRes);
 };
 
+export const emptyRefToSimpleName = (ref: string) => {
+  const trimmed = ref.replace(WEAVE_REF_PREFIX_EMPTY, '');
+  try {
+    return trimmed.split('/')[1].split(':')[0];
+  } catch (e) {
+    return trimmed;
+  }
+};
+
```

Comment:
simple parse for our empty ref that goes from the fake URI to just the object_id